app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/BusStop.java

```diff
@@ -0,0 +1,65 @@
+package ithaca_transit.android.cornellappdev.com.ithaca_transit.Models;
+
+import android.content.Context;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.R;
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.Singleton.Repository;
+
+public class BusStop {
+    private Double latitude;
+    private Double longitude;
+    private String stopName;
+
+    public BusStop() {
```

Comment:
do we need to define this, if it doesn't do anything? don't all objects inherit the `()` constructor, unless I'm mistaken?